src/app/pages/connect/connect-routing.module.ts

```diff
@@ -7,6 +7,10 @@ const routes: Routes = [
   {
     path: '',
     component: ConnectPage
+  },
```

Comment:
I don't think this entry is necessary... try removing these four lines, and see if it still works. If so, you can leave them out. Otherwise, I was tripping, and you can leave them in.

Reply:
No can do. If these four lines are removed, pressing the connect button on the home page does not navigate me to the connect page. It appears these four lines are necessary.